.gitignore

```diff
@@ -8,6 +8,7 @@ config.log
 *.cfgc
 *.pyc
 .cache
+pytest_session.txt
```

Comment:
This file is annoying. It is created solely when you run `pytest` yourself, not as part of `scons`. For now we have to ignore it.